Description: Fix POD errors in urxvt manpage
 Fixes various warnings from POD. This patch also includes a fix from Gregor
 Herrman <gregoa@debian.org> fixing a POD error which was causing rxvt-unicode
 to FTBFS
Author: Ryan Kavanagh <rak@debian.org>
Author: gregor herrman <gregoa@debian.org>
Bug-Debian: http://bugs.debian.org/708026
Origin: vendor
Forwarded: no
Applied-Upstream: no
Reviewed-by: Ryan Kavanagh <rak@debian.org>
Last-Update: 2013-03-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: rxvt-unicode/doc/rxvt.1.pod
===================================================================
--- rxvt-unicode.orig/doc/rxvt.1.pod	2013-05-26 18:08:37.384696431 -0400
+++ rxvt-unicode/doc/rxvt.1.pod	2013-05-26 18:08:37.376696226 -0400
@@ -1207,8 +1207,12 @@
 
 Turn on/off ISO 14755 5.2 mode (default enabled).
 
+=back
+
 =head1 BACKGROUND IMAGE OPTIONS AND RESOURCES
 
+=over 4
+
 =item B<-pixmap> I<file[;oplist]>
 
 =item B<backgroundPixmap:> I<file[;oplist]>
Index: rxvt-unicode/doc/rxvt.7.pod
===================================================================
--- rxvt-unicode.orig/doc/rxvt.7.pod	2013-05-26 18:08:37.320694794 -0400
+++ rxvt-unicode/doc/rxvt.7.pod	2013-05-26 18:08:47.260948834 -0400
@@ -1037,8 +1037,6 @@
 
    xprop -root XIM_SERVERS
 
-=item
-
 =back
 
 =head3 My input method wants <some encoding> but I want UTF-8, what can I do?
